snakebids/tests/strategies.py

```diff
@@ -51,6 +51,42 @@ def bids_entity_lists(
     ).filter(lambda v: v != ["datatype"])
 
 
+@st.composite
+def input_config(draw: st.DrawFn) -> InputConfig:
+    filtered_entities = draw(st.one_of(st.lists(bids_entity()), st.none()))
+    filters = (
+        {
+            entity.entity: draw(
+                st.one_of(st.booleans(), bids_value(), st.lists(bids_value()))
+            )
+            for entity in filtered_entities
+        }
+        if filtered_entities is not None
+        else None
+    )
+    wildcard_entities = draw(st.one_of(st.lists(bids_entity()), st.none()))
```

Comment:
```suggestion
    wildcard_entities = draw(st.one_of(st.lists(bids_entity()).map(str), st.none()))
```

This will eliminate the need for the below comprehension